### DISCUSSION TITLE
Uploading with phyton

### DISCUSSION THREAD

**Discussion by ghost** [community/community#23710] [Other Feature Feedback, Questions, & Ideas]
<p>Can you use a code to upload images using phyton in a github website?</p>

**Comment by otter-computer** [comment]
<p>I’m not quite sure what you mean by that. Can you explain a little more exactly what you’re trying to achieve?</p>

**Comment by R4356th** [comment, marked as answer]
<p>GitHub Pages cannot be used for dynamic websites. Here is a link for reference-</p><aside class="quote" data-post="4" data-topic="116096">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/hosting-wikis-with-github-pages/116096/4">Hosting Wikis With GitHub Pages</a> <a class="badge-wrapper  bullet" href="/c/github-help/github-pages/31"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #c8e1ff;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Discussion and questions about GitHub Pages—websites for you and your projects hosted right on your GitHub repository.">GitHub Pages</span></a>
  </div>
  <blockquote>
    Hello <a class="mention" href="/r4356th">@R4356th</a>, welcome to the community. 
No, GitHub Pages is for hosting of static content only. In other words, if you want to change what’s on the website, you have to upload new HTML to the repository. Dynamically-generated content, like wikis (including Mediawiki), is not supported on GitHub Pages.
  </blockquote>
</aside>
<p>
.</p>

**Comment by ghost** [comment]
<p>Look at <a class="mention" href="/r4356th">@R4356th</a>’s message<br>
And thanks for helping :wink:</p>
